### PR TITLE
kustomize 3.0.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
21:29:43 ==> /usr/local/Cellar/kustomize/3.0.3/bin/kustomize version
21:29:43 ==> /usr/local/Cellar/kustomize/3.0.3/bin/kustomize build /private/tmp/kustomize-test-20190718-13714-qm2fwq
21:29:43 Error: json: cannot unmarshal string into Go struct field Kustomization.patches of type types.Patch
21:29:43 Error: kustomize: failed

**Comment by darklore** [comment]
`patches:` in test has been removed from deprecated field?
https://github.com/kubernetes-sigs/kustomize/commit/a85f297f3147ef0aeaae32b15de54221d8410af6#diff-0144cdc603a417dd999ebfba9a9062bc

So that it should be renamed to `patchesStrategicMerge`?

https://github.com/kubernetes-sigs/kustomize/commit/b713d5a1cc6c221d26c7c4d77c11ca189ead2e77

**Comment by chenrui333** [comment]
@darklore Thanks for pointing out the needed change. I have made the change and passed my test locally.

File: Formula/kustomize.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Kustomize < Formula
   desc "Template-free customization of Kubernetes YAML manifests"
   homepage "https://github.com/kubernetes-sigs/kustomize"
   url "https://github.com/kubernetes-sigs/kustomize.git",
-      :tag      => "v3.0.2",
-      :revision => "aa2313c2825a7712b12309c1cd7798f371a0bb18"
+      :tag      => "3.0.3",
```
Comment (mdaniel) [inline Formula/kustomize.rb]: I don't know why the `v` was erroneously dropped from the version, nor how this made it out of CI, but [one can see it is v3.0.3](https://github.com/kubernetes-sigs/kustomize/releases) and the typo causes `brew install -s kustomize` to fail

Comment (chenrui333) [reply Formula/kustomize.rb]: yeah, I think the release tag was wrong earlier.

**Pull request closed without merging**